src/bindings/python/src/pyopenvino/core/offline_transformations.cpp

```diff
@@ -43,6 +44,9 @@ void regmodule_offline_transformations(py::module m) {
         "apply_moc_transformations",
         [](std::shared_ptr<ov::Model> function, bool cf) {
             ov::pass::Manager manager;
+            auto gr = manager.register_pass<ngraph::pass::GraphRewrite>();
```

Comment:
In this case we will apply this transformations to legacy pipeline too.
Let's move this transformation to separate function and enable only for new FEs.

I think that it is dangerous to switch compression logic in MO right before FF.
@jane-intel FYI ^^^